zappa/wsgi.py

```diff
@@ -20,31 +20,68 @@ def create_wsgi_request(
     binary_support=False,
     base_path=None,
     context_header_mappings={},
+    event_version="1.0",
 ):
     """
     Given some event_info via API Gateway,
     create and return a valid WSGI request environ.
     """
-    method = event_info.get("httpMethod", None)
-    headers = merge_headers(event_info) or {}  # Allow for the AGW console 'Test' button to work (Pull #735)
-
-    # API Gateway and ALB both started allowing for multi-value querystring
-    # params in Nov. 2018. If there aren't multi-value params present, then
-    # it acts identically to 'queryStringParameters', so we can use it as a
-    # drop-in replacement.
-    #
-    # The one caveat here is that ALB will only include _one_ of
-    # queryStringParameters _or_ multiValueQueryStringParameters, which means
-    # we have to check for the existence of one and then fall back to the
-    # other.
-
-    if "multiValueQueryStringParameters" in event_info:
-        query = event_info["multiValueQueryStringParameters"]
-        query_string = urlencode(query, doseq=True) if query else ""
-    else:
+    if event_version == "2.0":
+        method = event_info["requestContext"]["http"]["method"]
```

Comment:
Can you add a link to the aws documentation about the v 2.0 format here?

Reply:
done